www_src/pages/project/project.jsx

```diff
@@ -212,7 +213,33 @@ var Project = React.createClass({
       }
 
     });
+
+    // Handle button actions on zoomed in pages
+    dispatcher.on('linkClicked', function (event) {
```

Comment:
it's an ES6 thing; arrow functions bind to the parent context, so they inherit both `this` and `arguments`